Code/fbuild.bff

```diff
@@ -660,6 +660,7 @@ Alias( 'All+Tests' )
 	.Targets		= { 'All', 'Tests' }
 }
 
+#if __WINDOWS__
```

Comment:
This shouldn't be necessary. The latest version (v0.87) supports VS project and solution generation on all platforms.